asm/overlay_16_data.s

```diff
@@ -0,0 +1,7 @@
+	.public unused_02201BA0
```

Comment:
I put this in its own file for now because I don't know how to make it match in the C file. If I defined it in the C file, it would end up in .bss instead of .data. It will go to .data if I initialize it to a non-zero value, but then... it's not a match because it's suppposed to be zero.